L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -210,7 +210,7 @@ void TTStubBuilder<Ref_Phase2TrackerDigi_>::produce(edm::Event& iEvent, const ed
 
       /// Here tempOutput stores all the stubs from this lower cluster
       /// Check if there is need to store only one (if only one already, skip this step)
```

Comment:
Should we also update the commented line accordingly?

Reply:
@qliphy thanks for the comment which actually pointed me toward a small bug in the code. I will repush a version with comments and the fix (which is just one line the loop). So please wait for the update before pushing the PR

Reply:
I'm currently running some quick checks but the impact should be negligible. The case where there is more than 2 stubs formed from one seed in a PS module was not accounted correctly (only 1 stub was kept). This should be very rare anyway, but better to implement the exact behavior